integration_test/input-validation.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I am not seeing where this file is called or utilized to validate variable. Shouldn't this be added to the setup files?

Reply:
bamboo.yaml, line 321